src/microbe_stage/MicrobeStage.cs

```diff
@@ -566,6 +566,11 @@ public void MoveToMulticellular()
             throw new Exception("failed to keep the current scene root");
         }
 
+        var endosymbiosis = GameWorld.PlayerSpecies.Endosymbiosis;
+
+        if (endosymbiosis.StartedEndosymbiosis != null)
+            endosymbiosis.CancelEndosymbiosisTarget((int)endosymbiosis.StartedEndosymbiosis.Species.ID);
```

Comment:
I think it would be better to write a new method in EndosymbiosisData called something like `CancelAllTargets` (or something similar) that would just always be called here, and that would then clear up any endoymbiosis targets. That way the code here will keep working if ever it is added support for to have multiple ongoing endosymbiosis processes at once.